utilities/cli/container.py

```diff
@@ -627,6 +615,20 @@ def get_volume_args(self, add_volumes: List[str], enable_mps: bool) -> List[str]
             else:
                 print("Warning: MPS directories not found. MPS may not be enabled on the host.")
 
+        # sccache mounting
+        _, enable_sccache = get_env_bool("HOLOHUB_ENABLE_SCCACHE", default=False)
```

Comment:
Do we need/want a flag for this (instead of just env)? I feel like no since this is for app buildtime not runtime so don't want to forget it on some docker run calls

Reply:
Cc @tbirdso @wyli